amd_openvx_extensions/amd_rpp/source/tensor/SequenceRearrange.cpp

```diff
@@ -101,46 +99,46 @@ static vx_status VX_CALLBACK processSequenceRearrange(vx_node node, const vx_ref
     if (data->deviceType == AGO_TARGET_AFFINITY_GPU) {
 #if ENABLE_OPENCL
         cl_command_queue handle = data->handle->cmdq;
-        for (int sequence_cnt = 0; sequence_cnt < data->srcDescPtr->n; sequence_cnt++) {
-            unsigned src_sequence_start_address = sequence_cnt * data->srcDescPtr->strides.nStride * data->sequenceLength;
-            unsigned dst_sequence_start_address = sequence_cnt * data->dstDescPtr->strides.nStride * data->newSequenceLength;
+        for (int sequence_cnt = 0; sequence_cnt < data->pSrcDesc->n; sequence_cnt++) {
```

Comment:
Can unsigned be used for this loop variable?

Reply:
Changed in the other PR
https://github.com/fiona-gladwin/MIVisionX/commit/816870a1674aaceffc34568aa87f9f9745130289